springwolf-examples/springwolf-kafka-example/src/main/avro/ExamplePayloadAvroDto.avsc

```diff
@@ -1,26 +1,38 @@
 {
+    "namespace": "io.github.stavshamir.model",
```

Comment:
Is it best practice to generate the avro models to a different package?
Leaving it at `io.github.stavshamir.springwolf.example.kafka.dto.avro` does reduce the diff size - also in the asyncapi.json